GitUI/BranchTreePanel/RepoObjectsTree.Nodes.cs

```diff
@@ -242,7 +242,7 @@ protected internal virtual void Refresh()
             /// <param name="isFiltering">
             ///  <see langword="true"/>, if the data is being filtered; otherwise <see langword="false"/>.
             /// </param>
-            internal void ToggleFilterMode(bool isFiltering)
+            internal void Refresh(bool isFiltering, Func<RefsFilter, IReadOnlyList<IGitRef>> refs)
```

Comment:
❔ The argument is not used (in this non-virtual function).

Reply:
As above